### ISSUE TITLE
Expose Session expiry time? 

### ISSUE THREAD

**Issue by moddyz** [pogodevorg/pogo#14]
I was looking at the code in this repo https://github.com/AHAAAAAAA/PokemonGo-Map and it looks like there each provider has a expiry time that can be queried.  
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/master/pogom/pgoapi/pgoapi.py#L178
Upon expiry it looks like the client relogs in.

It looks like it's part of the AuthTicket struct:
https://github.com/pkmngo-odi/pogo-protos/blob/master/networking_envelopes.pb.go#L48

Could we possibly expose that on the session object so we know when to login again?  

Do let me know if I'm not understanding Session properly :)

**Comment by zeevallin** [comment]
I agree, session should provide an interface for forcefully refreshing tokens and expose functions for checking current expiry times. I'll probably work on this next.

**Comment by femot** [comment]
Part of it implemented here: https://github.com/femot/pgoapi-go/blob/master/api/session.go#L67
